packages/wallet/src/redux/actions.ts

```diff
@@ -193,3 +193,20 @@ export {
   isCommonAction,
   application,
 };
+
+// These are any actions that update shared data directly without any protocol
+export type SharedDataUpdateAction = AdjudicatorEventAction;
+
+export function isSharedDataUpdateAction(action: WalletAction): action is SharedDataUpdateAction {
+  return isAdjudicatorEventAction(action);
+}
+
+export function isAdjudicatorEventAction(action: WalletAction): action is AdjudicatorEventAction {
+  return (
+    action.type === CONCLUDED_EVENT ||
```

Comment:
We could filter based on the `WALLET.ADJUDICATOR` prefix. This approach would allows us to add new adjudicator events without updating the filter.